server/controllers/shoppingList.js

```diff
@@ -194,9 +208,13 @@ const getListData = (req, resp) => {
         ? { _id, cohortId, description, isAdmin, isArchived, name, products }
         : { _id, cohortId, description, isArchived, name, products };
```

Comment:
in previous PR this was diffrent, as I said earlier you can send isAdmin with value false or true and this ternary will be not need